packages/shared/lib/core/account/actions/findBalances.ts

```diff
@@ -16,11 +18,13 @@ let searchAccountStartIndex: number
 let _accountGapLimit: number
 let _addressGapLimit: number
 
+let profileType: ProfileType
+
 export function initialiseAccountRecoveryConfiguration(
     algortihmType: SearchAlgorithmType,
     config?: RecoverAccountsPayload
 ): void {
-    const profileType = get(activeProfile)?.type
+    profileType = get(activeProfile)?.type
```

Comment:
Let's try not to use global functions to store state between functions. If you want to get the profile Type in `findBalances` just call `get` on the store again.